reference/docs-conceptual/setup/PowerShell-Core-on-ARM.md

```diff
@@ -0,0 +1,12 @@
+# PowerShell Core on ARM
```

Comment:
When this location change gets merged to live, https://aka.ms/pscore6-arm needs to be updated to point to the new location. I've given permission for writers to edit that link in my absence (@SteveL-MSFT and @SandeepSutari also have permission there if anything goes awry).

Reply:
This is the only one as far as I'm aware, but I'll double check with @SteveL-MSFT when I get back.